drivers/bridge/bridge.go

```diff
@@ -1019,6 +1026,27 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		}
 	}
 
+	// Add a neighbor proxy if using NDP proxying
+	if config.NDPProxyInterface != "" && config.EnableIPv6 {
+		link, err := d.nlh.LinkByName(config.NDPProxyInterface)
+		if err != nil {
+			return err
```

Comment:
Simply `return fmt.Errorf("could not find link for ndp proxy interface %q: %v", config.NDPProxyInterface, err)`